wordpress/wp-content/themes/postlight-headless-wp/inc/admin.php

```diff
@@ -46,6 +46,9 @@ function set_headless_preview_link( $link ) {
 
 add_filter( 'preview_post_link', 'set_headless_preview_link' );
 
+add_filter( 'post_link', 'set_headless_preview_link' );
```

Comment:
I need to check this closer - so I'll run the project, but it looks like you're setting the link of normal posts and pages as a preview link. No action needed here for now, but I gonna check few things